src/bioregistry/align/ncbi.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+"""Align NCBI with the Bioregistry."""
+
+from typing import Any, Dict, List
+
+from bioregistry.align.utils import Aligner
+from bioregistry.external.ncbi import get_ncbi
+
+__all__ = ["NcbiAligner"]
+
+
+class NcbiAligner(Aligner):
+    key = "ncbi"
+    getter = get_ncbi
```

Comment:
if you add `curation_header = ('name',)` then it knows how to pretty print the curation table better